cypress/e2e/event-info-spec.cy.js

```diff
@@ -0,0 +1,40 @@
+describe('Event Info', () => {
+  beforeEach(() => {
+    cy.login();
+    cy.getEventOne();
+    cy.clickEventOne();
+  });
+
+  it('Should have the correct url', () => {
+    cy.url().should('include', '/events/1');
+  });
+
+  it('Should have a visible header', () => {
+    cy.get('header').should('be.visible');
+  });
+
+  it('Should have a title for the event', () => {
+    cy.get("[data-cy='event-title']").should('be.visible').should('contain', 'Party at the Park');
```

Comment:
great work using data attributes instead of relying on tag names, ids, or classes